components/BookingOverview/index.js

```diff
@@ -0,0 +1,50 @@
+import Link from "next/link";
+import { HorsesList } from "@/lib/data";
+import { ImageCowgirlOnWhiteHorse } from "../../Styles/ImageCowgirlOnWhiteHorse";
+import { CardStyled } from "@/Styles/CardStyled";
+
+import { DeleteButton } from "@/Styles/Buttons";
+
+export default function BookingOverview({
+  recentlyBooked,
+  timeSlots,
+  handleDelete,
+}) {
+  console.log("Delete:", handleDelete);
+
+  return (
+    <>
+      <CardStyled>
+        <ImageCowgirlOnWhiteHorse />
+        <p>Datum: {recentlyBooked?.currentDate}</p>
+        <p>Uhrzeit: {recentlyBooked?.startTime}</p>
+        <div>Personen: {recentlyBooked?.numberOfPeople}</div>
+        <p> Pferde:</p>
+        <ul>
+          {recentlyBooked?.horses.map((horseID) => (
+            <li key={horseID}>
+              {HorsesList.find((horse) => horse.id === horseID).name}
```

Comment:
I think it would be more efficient if you would filter your HorseList and then map over it, but this solution is fine as well.